### ISSUE TITLE
Rebrand Collective App

### ISSUE THREAD

**Issue by SamirSaidani** [nextcloud/collectives#576]
First, _I'm not questioning the choice you've made for the name, Collectives is a nice one,_ and I know that a lot of discussion was involved in the naming choice.

I'm just asking for freedom on the choice of the name, like already suggested in other issues (see #41 #189).

We would like for internal purpose and clarity rebrand the Collective Apps with _OurWiki_ (actually we want to keep the name of our previous wiki system to make the transition easier for our users) and optionally change the logo. We want to use Collective Apps as an internal/shareable source of documentation.

For us, renaming would give more clarity for our end-users : we have already File, Tasks, Agenda, Contact Apps whose name are pretty straightforward, and we would like to keep this kind of consistency.

A way to do it would be exactly like rebranding Nextcloud:
- as admin, go to Settings, Choose Customize Collectives, change Name, and Logo (see for example settings/admin/theming)

Thanks a lot for this awesome app, we are seriously considering to replace our entire current wiki system with Collective Apps.

**Comment by max-nextcloud** [comment]
@SamirSaidani I think I get your usecase. I'm wondering if it can be achieved in a more generic way that's not specific to the collectives app.

Just as a thought experiment for now... what if you:
* Create a collective called *Grimoire* to contain your wiki systems content.
* Install the [external sites app](https://apps.nextcloud.com/apps/external)
* Add an "external" site pointing to that collective with the icon and name you like

That way the link would not just link to the list of collectives but directly to the content of the wiki shared with every user. (Assuming here that your current wiki does not have different parts for different groups of users).

You might still want to get rid of the collectives link (in case you don't want users to create their own collectives for workgroups or so). I had a brief look at the appstore but i did not find a way to hide apps from the topmenu. Customizing CSS might be an option but that feels hacky.

Anyway... just brainstorming here to see if there's a more generic solution that we could work on that would also benefit users wanting to customize other apps.

**Comment by SamirSaidani** [comment]
Thanks max for your answer.

I'm totally ok with the way Collectives work, the ability for a Circle to build its own wiki: the solution you provide seems convoluted compared with just rebranding the name/logo of Collective.

I've started to dig with the l10n file to see if I just can change the name here, by overriding the theme and l10n: it would be a hackish way though.

I agree that it's interesting to look for more generic solutions, maybe we could have at Nextcloud level a way to change the name for an app: but in the case of Collectives, it's not that simple, because the app use the term Collective not only for the name, but also as a term to create circle documentation inside the app. 

For Collectives, I think that specific rebranding app setting is necessary, to spread the change of name across the app. Instead of "create a new Collective", you would have "create a new Wiki/CircleDoc/Grimoire/WhatEverYouWant".

**Comment by max-nextcloud** [comment]
@SamirSaidani You're right... there's quite a few places to customize. Also in the files extension and so on.

I think the l10n files are actually the right place to do this in particular since you only need this in one language i assume.

The intersection with translations would also be a problem for coming up with a way to substitute `collective` with another term. For example in german if you turn "Kollektiv" into "Kreis" (circle) "Neue*s* Kollektiv" becomes "Neue*r* Kreis". I think it's the same in french "nouvelle documentation" vs "nouveau collectif", right?

So I think customizing the locale is actually the most promising approach - I think you can also do this in a theme - but i have not tried it myself. See https://help.nextcloud.com/t/how-to-change-the-welcome-message-and-translation-for-the-welcome-email-sent-to-a-new-user/135811/8

**Comment by SamirSaidani** [comment]
My initial feature request was to simplify the life of nextcloud admin to set a new name directly from app settings. I've noticed that a special directory named "Collective" is created for each user. I doubt that changing locale would change the Collective user directory name. Maybe it's the case ? But if it's not, it means that even if I manage to change the name through locale, the user will be confused with a directory name not matching the app name.

To be consistent for "rebranding" the app, I need to change the name in:
- the name of the app itself, as it appears in the main nextcloud menu,
- inside the app,
- the directory created by the app for each user.

I think that it would be more consistent and far more simple for a nextcloud admin to do it in the app settings.

**Comment by SamirSaidani** [comment]
I would go for Wiki.

It already conveys the idea of collective of equals, and the whole free culture philosophy (cf Wikipedia).

I'm totally into the idea of collective of equals, I'm representing a democratic collective of freelancers (coop). 

Maybe we can think of it in another way. 

Collective could be a nextcloud distribution, a suite of nextcloud apps designed to empower a collective :
- Wiki app to share knowledge 
- Gouv app to describe the roles and responsibilities into the collective 
- Meeting app to support collective meeting with shared agenda items (see holaspirit meetings for instance)

**Comment by marcoambrosini** [comment]
I think it's important to consider the vision for the app before renaming. If we're going in the direction of Notion, Microsoft loop, anytype etc. it would be reductive to call it either wiki or knowledge. 
It's precisely the looser relation with what the app does that the name "collectives" allows us to play around a bit more with what we actually want to do there.
E.g. If we call it wiki, and we introduce a task or table view via smart picker, well we suddenly have a wiki that can be used as a project management tool. In such case, we'd really have a name that doesn't work.

**Comment by nimishavijay** [comment]
On the topic of name changes, one thing to note is that currently it's not clear to new users what the name of the app even is, as 2 participants in the design feedback calls mentioned "Collections" when referring to the Collectives app. 
In the app bar on top, Collectives is ellipsised to "Collec..." and at that point there's no way a user can tell it's called Collectives unless they navigate into the app. So I would suggest to also keep in mind the length of the new name in case a rebranding is on the table.

**Comment by SamirSaidani** [comment]
> E.g. If we call it wiki, and we introduce a task or table view via smart picker, we suddenly have a wiki that can be used as a project management tool. In such case, we'd really have a name that doesn't work.


This is already true with MediaWiki; just look at the numerous plugins available. Modern wikis have evolved far beyond what they used to be, but their essence remains the same :  be able to change the content fast and collaboratively, whatever the content is. 

Although I really like the term "Collective" and the inspiration behind it, I've learned to prioritize readability over creativity. I believe "Wiki" aligns with Nextcloud's official app naming policy, as I understand it: choosing short and clear names for official apps, such as File, Photo, Tasks, etc.
